api/scripts/fetch_feature_layer.py

```diff
@@ -0,0 +1,98 @@
+# Taken from:
+# https://support.esri.com/en/technical-article/000019645
+# Minus the token stuff
+
+# referenced https://developers.arcgis.com/rest/services-reference/enterprise/query-feature-service-layer-.htm
+
+
+import urllib.parse
+import urllib.request
+import os
+import json
+# import arcpy
```

Comment:
Kill the stray code comments in this file, and the prints (or replace them with logger statements)

Reply:
i.m.h.o this entire file belongs in a separate repository. wps_tileserver ; i've commited it up just for reference, but it's totally not ready. there are some features that fail import and that has to be addressed.
also - it needs all kind of instructions. e.g. gdal-bin needs to be installed, and how to run it directly against a database in the openshift cluster - as I've been doing.